keys/constants.go

```diff
@@ -117,6 +117,11 @@ var (
 	// LocalMax is the end of the local key range.
 	LocalMax = LocalPrefix.PrefixEnd()
 
+	// TableDataPrefix prefixes all Table data to aid in transitioning
+	// key:value data to Table data, and for ease of debugging.
+	TableDataPrefix       = proto.Key("table-")
+	TableDataPrefixLength = 6
```

Comment:
Seeing how this value is used, I'd just nuke it and use `len(TableDataPrefix)` when needed.